### ISSUE TITLE
Logical Structure/Meaningful Sequence

### ISSUE THREAD

**Issue by rosejbon** [neontribe/ca-hospice-hope#80]
## Description
 
The layout of pages is conducive to a user seeking support. HTML5 use is generally semantic. A few potential exceptions are documented here:
    - [ ] Use of headings for long Call To Action text can make the outline that a screenreader would use bloated. e.g.
    ![image](https://user-images.githubusercontent.com/2150762/63609898-02a8d800-c5d0-11e9-94a8-f60f57b8499a.png)
   - [x] Some sections, while made distinct with visual styling and HTML5 `<section>` tags, are left without headings. This leaves them invisible and unlabelled in the document outline. For each section, it should be considered as to whether they would be deserving of a place in the outline. e.g.
      * The information given after a story:
        ![image](https://user-images.githubusercontent.com/2150762/63610714-fd4c8d00-c5d1-11e9-91b9-f6bf62d05759.png)
      * The dates given within "When do sessions take place?"
 - [x] Some areas may benefit from being explicitly ordered or marked as unordered, to aid screenreader navigation features. e.g. the footer is only sectioned with <div> elements
    ![image](https://user-images.githubusercontent.com/2150762/63610219-d3469b00-c5d0-11e9-873b-223cccc410b5.png)

**Comment by rosejbon** [comment]
To add issues that are ticked need adding to main contemplating action repo probably to the backlog

**Comment by rosejbon** [comment]
Closing because have added to main repo. We are not worried about the first point as the copy is meant to be alot shorter, we had to lengthen it just for this project.
